alphagov/paas-billing
# Remove minimum 1p charge for billable event

# What

The 1p charge is unfairly penalising tenants who do very large numbers
of very short events.

This Reverts the decision in commit 04a0852 where we added a minimum charge
of 1p to billable events.

The minimum charge was added at the time to simplify calculations where
we had grouped/summed usage and due to uncertainty around how to
calculate VAT on fractions of a penny. We now know that keeping the
prices acurate to at least 5decimal places is suitable for calculating
VAT so it should now be safe to remove.

It is expected that this will cause the bills to lower slightly.

# How to review

* Code review
* Check tests
* Create a paas-cf bump PR after merging

# Who can review

Not @chrisfarms
